rapid7_insightidr/plugin.spec.yaml

```diff
@@ -494,3 +494,53 @@ actions:
         description: Events from logs
         type: "[]events"
         required: true
+  close_investigations_in_bulk:
+    title: Close Investigations in Bulk
+    description: Close all investigations that match the parameters
+    input:
+      source:
+        title: Source
+        description: The name of an investigation source
+        type: string
+        example: MANUAL
+        default: MANUAL
+        required: false
+        enum:
+          - ALERT
+          - MANUAL
+          - HUNT
+      datetime_from:
```

Comment:
We should set the default with `default` param so users can easily modify

Reply:
Here we must use some datetime. When I set default parameter to: "2020-09-01" it might be confuse for user. In code it is set for one last week. But as default I think we shouldn't set any value.

Reply:
hmm, if we set it to a default timestamp for one week ago, it's easier for the user to see that and change it I think. My concern is the date format is complex for users, so less error to have them edit a field than to fully type it in when it looks like this: `2018-07-01T00:00:00Z`.

@mrinehart-r7 What are your thoughts?